appcircle_filesize/1.0.1/component.yaml

```diff
@@ -0,0 +1,37 @@
+platform: Common
+buildPlatform:
+displayName: File size check
+description: "Checks the file size of build artifacts."
+docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps#file-size-check
+requiredComponents: "appcircle_ios_build_sign,appcircle_android_build"
+precedingComponents: 
```

Comment:
Other build steps that give app output should also be added here.

Reply:
on the ios side only appcircle_ios_sign component generates ipa that user can use. The other generating ipa steps are related to 3rd party test automation tools and user can not use directly to them.